flink-filesystems/flink-s3-fs-presto/pom.xml

```diff
@@ -272,12 +272,6 @@ under the License.
 										<exclude>META-INF/LICENSE.txt</exclude>
 									</excludes>
 								</filter>
-								<filter>
-									<artifact>com.facebook.presto.hadoop:hadoop-apache2</artifact>
-									<includes>
-										<include>com/facebook/presto/hadoop/**</include>
-									</includes>
-								</filter>
```

Comment:
Why are these changes necessary?

Reply:
eh... whoops. these don't belong in here, reverting.